Userland/Applications/Calculator/Keypad.cpp

```diff
@@ -136,3 +134,8 @@ String Keypad::to_string() const
 
     return builder.to_string();
 }
+
+void Keypad::set_rounding_length(unsigned rounding_threshold)
+{
+    m_displayed_frac_length = rounding_threshold;
```

Comment:
We usually prefer using full names over abbreviations - `m_displayed_fraction_length`